tests/conftest.py

```diff
@@ -10,6 +10,10 @@
 from .basic_env import TestDirFixture
 
 
+# Prevent updater and analytics from running their processes
+os.environ["DVC_TEST"] = "true"
```

Comment:
Maybe `tests/__init__.py` would be even better?

Reply:
This looks like more obvious location to me. E.g. autouse fixtures are here.

Reply:
So this didn't work for you before, because you didn't run tests through `python -mtests`?

Reply:
Yes, I generally don't want to use 4 processes and verbose mode.